flavors.yaml

```diff
@@ -0,0 +1,475 @@
+---
+targets:
+  - name: ali
+    category: public-cloud
+    flavors:
+      - modifier: gardener_prod
+        features: []
```

Comment:
we should really rethink this split `modifier` - `features` in here, should ideally only be `features` and maybe a modifiers (but also as array) entry on the level above it to uniformly apply to all flavours within the same name

Reply:
We should clarify if `modifier` is really needed, yes.
If nothing speaks against it, we should just add the modifier to features likes this:
```
features:
  - gardener
  - _prod
  - _trustedboot
  - _tpm2
  ```

This would simplify a lot of logic that does modifier/feature parsing.